src/client/render/pipeline.cpp

```diff
@@ -140,7 +140,8 @@ bool TextureBuffer::ensureTexture(video::ITexture **texture, const TextureDefini
 	if (definition.valid) {
 		if (definition.clear) {
 			video::IImage *image = m_driver->createImage(definition.format, size);
-			image->fill(0u);
+			// Cannot use image->fill because it's not implemented for all formats.
+			std::memset(image->getData(), 0, image->getDataSizeFromFormat(definition.format, size.Width, size.Height));
```

Comment:
that sounds wrong, which format do we use that Irrlicht can't deal with?

Reply:
video::ECF_A16B16G16R16F for all textures dealing with HDR color, if supported.

If you look at CImage, it only supports 8-bit and 565-bit RGB/ARGB formats.

Reply:
Then wouldn't a fix in IrrlichtMt be a saner solution?

Reply:
Yes, it's a stop-gap solution for all uncompressed formats. I plan to move it to irrlicht after the release.

Reply:
Irrlicht was already tagged so changing it now would be inconvenient, so sounds good.